images/step-issuer/tests/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_providers {
+    oci       = { source = "chainguard-dev/oci" }
+    imagetest = { source = "chainguard-dev/imagetest" }
+  }
+}
+
+variable "digest" {
+  description = "The digest of the image"
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+
+variable "target_repository" {
+  description = "The docker repo name"
+}
+
+data "imagetest_inventory" "this" {}
+
+resource "imagetest_harness_k3s" "this" {
+  name      = "step-issuer"
+  inventory = data.imagetest_inventory.this
+
+  sandbox = {
+    envs = {
+      "IMAGE_REGISTRY"   = data.oci_string.ref.registry
+      "IMAGE_REPOSITORY" = data.oci_string.ref.repo
+      "IMAGE_TAG"        = data.oci_string.ref.pseudo_tag
+    }
+    mounts = [
+      {
+        source      = path.module
+        destination = "/tests"
+      }
+    ]
```

Comment:
```suggestion
```

I don't think this is needed since there's nothing outside the terraform module being run. If there was a "test.sh" or similar script, then the mounts block would add it is a docker volume into the kubectl container.